warehouse/models/gtfs_views/reports_daily_service_routes_stops.sql

```diff
@@ -0,0 +1,132 @@
+{{ config(materialized='table') }}
+
+WITH gtfs_schedule_fact_daily_service AS (
+    SELECT *
+    FROM {{ ref('gtfs_schedule_fact_daily_service') }}
+),
+
+gtfs_schedule_fact_daily_feed_stops AS (
+    SELECT *
+    FROM {{ ref('gtfs_schedule_fact_daily_feed_stops') }} 
+),
+
+gtfs_schedule_fact_daily_feed_routes AS (
+    SELECT *
+    FROM {{ ref('gtfs_schedule_fact_daily_feed_routes') }} 
+),
+
+gtfs_schedule_dim_feeds AS (
+    SELECT *
+    FROM {{ ref('gtfs_schedule_dim_feeds') }}
+),
+
+dates AS (
+    SELECT full_date AS service_date
+    FROM {{ ref('dim_date') }}
+),
+
+reports_gtfs_schedule_index AS (
+    SELECT *
+    FROM {{ ref('reports_gtfs_schedule_index') }} 
+),
+
+stops_agg AS (
+    SELECT
+        feed_key,
+        date AS service_date,
+        COUNT(DISTINCT stop_key) AS n_stops
+    FROM gtfs_schedule_fact_daily_feed_stops
+    GROUP BY 1, 2
+),
+
+routes_agg AS (
+    SELECT
+        feed_key,
+        date AS service_date,
+        COUNT(DISTINCT route_key) AS n_routes
+    FROM gtfs_schedule_fact_daily_feed_routes
+    GROUP BY 1, 2
+),
+
+stops_routes_metadata_joined AS (
+    SELECT
+        T1.feed_key,
+        T1.n_stops,
+        T1.service_date,
+        T2.n_routes,
+        T3.calitp_itp_id,
+        T3.calitp_url_number
+    FROM stops_agg AS T1
+    INNER JOIN routes_agg AS T2
+        ON T1.service_date = T2.service_date
+            AND T1.feed_key = T2.feed_key
+    INNER JOIN gtfs_schedule_dim_feeds AS T3
```

Comment:
Is this the right way to combine those joins?

Reply:
Yes, looks good to me!